NetTopologySuite/Index/KdTree/KdTreeExtensions.cs

```diff
@@ -77,8 +77,11 @@ public static class KdTreeExtensions
             if (isOddLevel)
                 return (left ? target.X <= node.X : target.X >= node.X) || Math.Pow(target.X - node.X, 2) < closestDistSq;
             return (left ? target.Y <= node.Y : target.Y >= node.Y) || Math.Pow(target.Y - node.Y, 2) < closestDistSq;
+
+/*
```

Comment:
If it was there in the old version then leave it I would say.

We can cleanup code later, this is just about getting .NET Core support.